unlock-app/src/components/interface/VerificationStatus.tsx

```diff
@@ -58,52 +72,107 @@ export const VerificationStatus = ({ data, sig, hexData }: Props) => {
       sig,
     }) === accountAddress.toLowerCase()
 
+  const { providers, externalRefundContractAddress } = config
+  const provider = providers[Object.keys(providers)[0]]
+
   return (
     <div>
       <Identity valid={identityIsValid} />
 
       <OwnsKey
         loading={queryResults.loading}
         error={queryResults.error}
-        data={queryResults.data}
-        lockAddress={lockAddress}
+        matchingKey={matchingKey}
       />
 
       <p>
         Signed {durationsAsTextFromSeconds(secondsElapsedFromSignature)} ago.
       </p>
+      {matchingKey &&
+        identityIsValid &&
+        lockAddress.toLowerCase() ===
+          '0x0AAF2059Cb2cE8Eeb1a0C60f4e0f2789214350a5'.toLowerCase() && (
```

Comment:
This is not the right lock address. I'm not 100% sure how we want to handle this. This lock address works for local dev, but we'll want to make sure we have values for both staging and mainnet. I don't know if it's worth using an environment variable or just hardcoding in the config file since this is temporary

Reply:
I think it's fine to do an array of addresses?